services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -158,7 +157,7 @@ async def shutdown(self) -> None:
                     "Following observation tasks completed with an unexpected error:%s",
                     f"{bad_results}",
                 )
-        except asyncio.TimeoutError:
+        except TimeoutError:
```

Comment:
Q: were these two errors unified?
I did not expect the below
```python
>>> import asyncio
>>> asyncio.TimeoutError == TimeoutError
True
```

Reply:
yes. Please check comments from Ruff. that is a feature of python 3.11